engine/CMakeLists.txt

```diff
@@ -178,6 +178,8 @@ ENDIF(NOT POW_FUNCTION_EXISTS AND NOT NEED_LINKING_AGAINST_LIBM)
 IF (NOT CMAKE_BUILD_TYPE)
     SET(CMAKE_BUILD_TYPE "RelWithDebInfo" CACHE STRING "Release, RelWithDebInfo, Debug, Profiler" FORCE )
 ENDIF (NOT CMAKE_BUILD_TYPE)
+MESSAGE("** Build Type: ${CMAKE_BUILD_TYPE}")
```

Comment:
Letting folks know which build is being used is probably wise. Not sure why we didn't do this earlier.